src/constants.tsx

```diff
@@ -0,0 +1,17 @@
+/** Constant string literals in alphabetical order */
+// IDs and keys (user-invisible strings)
+export const FOLDER_ID_KEY: string = 'webmarkFolderId';
+export const LOAD_HERE_KEY: string = 'loadHere';
+export enum NotificationId {
```

Comment:
No message for "load successful." Are we not showing such thing?

Reply:
Actually I'll add this while adding notifications

Reply:
@jeongm-in I believe we don't need one for load because it's visually evident that the operation was successful.

Reply:
Well wouldn't it be more consistent? But okay it's not necessary.